dev-libs/zziplib/zziplib-0.13.71-r1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
Sorry if I asked you this before. Does Python 3.9 work?

Reply:
Works.  Added python3_9.